birdhouse/scripts/get-services-json.include.sh

```diff
@@ -1,15 +1,13 @@
 #!/bin/sh
 
 # default value in case of error or missing definitions
-export BIRDHOUSE_DEPLOY_SERVICES_JSON='{}'
-if [ -z "${ALL_CONF_DIRS}" ]; then
-  echo "No services in DEFAULT_CONF_DIRS and EXTRA_CONF_DIRS. SERVICES JSON list will be empty!"
-  return
-fi
 
 for adir in ${ALL_CONF_DIRS}; do
   [ -f "${adir}/service-config.json" ] || continue
   SERVICES="${SERVICES}$([ -n "${SERVICES}" ] && echo ',') $(cat "${adir}/service-config.json")"
 done
 
+if [ -z "${SERVICES}" ]; then
+  echo "No services in DEFAULT_CONF_DIRS and EXTRA_CONF_DIRS. SERVICES JSON list will be empty!"
```

Comment:
Could use the prefix as in https://github.com/bird-house/birdhouse-deploy/blob/22fc7c01a0279b2c98a9059b874dee5922bcac21/birdhouse/pavics-compose.sh#L74